docker_readme.txt

```diff
@@ -0,0 +1,5 @@
+Steps for Docker: 
+ 1. install docker or docker for windows 
+ 2. docker build . in the directory where docker file is located
+ 3. docker images to get image name (or set it when building)
+ 3. docker run -t -d -v /var/run/docker.sock:/var/run/docker.sock {Image Name}
```

Comment:
Are you sure it works? what triggers the python script?

Reply:
Please see the docker-readme. I left it to manually run from exec, since we anyhow have to specify the flags (dev, etc...)